ktor-network/jvmAndNix/test/io/ktor/network/sockets/tests/UDPSocketTest.kt

```diff
@@ -213,6 +215,18 @@ class UDPSocketTest {
                 }
             }
     }
+
+    @Test
+    fun testUdpConnect() = testSockets { selector ->
```

Comment:
this test failed twice in different builds and succeed it others. can you check it please?

Reply:
it looks like flaky, will take a look

Reply:
It looks like a different behavior in win API, will take a look

Reply:
fixed: you cannot bind and connect single hostname and port on win